backend/app/src/routes/users.js

```diff
@@ -215,6 +215,55 @@ module.exports = async function (fastify, opts) {
       return { holdings }
     }
   })
+  
+  fastify.route({
+    method: 'GET',
+    url: '/users/:user_id/allholdings',
```

Comment:
Not sure I love this URL, but I like the way it's nested under `/users/:user_id`